framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/AbpAspNetCoreMvcModule.cs

```diff
@@ -38,6 +40,10 @@ public class AbpAspNetCoreMvcModule : AbpModule
     {
         public override void PreConfigureServices(ServiceConfigurationContext context)
         {
+            DynamicProxyIgnoreTypes.IgnoreTypes.AddIfNotContains(typeof(ControllerBase));
```

Comment:
Add `DynamicProxyIgnoreTypes.Add(...)` method which internally uses the `IgnoreTypes.AddIfNotContains`.